1.preprocessing_data/README.md

```diff
@@ -2,52 +2,36 @@
-We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
+To correct for illumination issues within the CFReT pilot data, we use the Background method for all plates from the [CellProfiler illumination correction modules](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.4/modules/imageprocessing.html?highlight=illumination+correction#correctilluminationapply).
 
-Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).
+To confirm that IC was working on the images, we adjusted the brightness/contrast in CellProfiler to see if the contrast between objects and background increased and the illumination looked more even. 
 
-Being able to visualize illumination errors can be hard for some datasets.
-To test if illumination correction is necessary for this dataset, we brightened images using [Fiji](https://imagej.net/software/fiji/).
-Through this, we did observe that there was technical variation in illumination across the images (e.g vignetting), which is a common issue for Cell Painting data (Figure 1).
+## Calculate and save corrected images for all plates
 
-![Raw_to_Bright_fig](example_figs/Raw_to_Bright_fig.png)
+To perform illumination correction and save the corrected images, you will need to:
 
-> Figure 1. Image Comparison: Raw to Brightened. This figure displays the images for each channel in the same well/frame. The channel metadata (red) and Cell Painting assay stain for each channel (blue) is shown to the left of the images. The main indicator of illumination issues is seeing brighter areas, especially in the background. For example, in the `Mito` channel, you can see that on the left side, the area is brighter as compared to the top right.
+### Step 1: Open the CellProfiler GUI
 
-After using PyBaSiC (or BaSiCPy) to correct for illumination, it is hard to tell if there was any change made to the images just by looking at them unedited (Figure 2).
-
-![NotBrightened_Comparison.png](example_figs/NotBrightened_Comparison.png)
-
-> Figure 2. Comparison Between Unedited Raw and Corrected Images. This figure shows that it is very hard for the human eye to see the difference between the raw and corrected images without any editing.
-
-By brightening the images through, there is a noticable difference in illumination between the raw and corrected images where you can tell the illumination is more even in the corrected images (Figure 3)
-
-![Raw_to_Corrected_fig](example_figs/Raw_to_Corrected_fig.png)
+```bash
+# activate the conda env for the repo
+conda activate cfret_data
+# call CellProfiler to start the GUI
+cellprofiler
```

Comment:
Does launching `cellprofiler` here depend on the environment somehow? Are there any caveats here to consider with regards to ensuring it runs as expected?

Reply:
It does depend on the environment since you need to have CellProfiler installed from `pip`. If they are in their base and they don't have CellProfiler installed, then this won't work. This is why there is one main environment for them to use instead of make one per module.